.github/workflows/test-ssl.yml

```diff
@@ -0,0 +1,118 @@
+name: Test-SSL
```

Comment:
can we cover the SSL testing in the existing test suite run instead if it is even necessary to introduce tests for this?

Reply:
Hey, the normal testing suite doesn't have ssl for the postgres. This test uses a different postgres image: ghcr.io/infrastructure-as-code/postgres:11 which allows for ssl.

Since the ssl config is optional and does not have to be enabled, I think it should be kept as seperate test. Although I added the test to keep in line with the pattern, unsure if its entirely necessary to have workflow test for ssl connection.

Reply:
After thinking about this for a bit, I would refrain from testing the actual ssl connectivity in the test suite altogether as this is more of a database level implementation detail, rather than something the metaflow-service has to care about.

For testing it should be enough to cover that a correct dsn is created from a DBConfiguration, when SSL related environment variables are present. It is up to the client library and DB then how they choose to handle the connection string.

Reply:
After thinking about this for a bit, I would refrain from testing the actual ssl connectivity in the test suite altogether as this is more of a database level implementation detail, rather than something the metaflow-service has to care about.

For testing it should be enough to cover that a correct dsn is created from a DBConfiguration, when SSL related environment variables are present. It is up to the client library and DB then how they choose to handle the connection string.

Reply:
Sounds good